Aligning and centering forms

tbentham opened the pull request:
Changes:
* Centered `login` and `register` forms based on Tailwind UI components.
* Aligned `member_details` and `member_settings` forms using CSS grids. (Resolves #165)
* General styling improvements to forms, e.g. font weights, header sizes, margins and padding.
* Fixed issue with invisible help text going off the right of the page and affecting page width in mobile view.

Non-form related:
* Adjusted logo font as I noticed at the last meeting that it wasn't working in Safari (I'm unable to test this on Windows, but should be working better now).

Freshrojek reviewed (approved):
Like what you've done, good job! especially to the registration form - looks much more classy

tbentham commented:
Thanks! Sorry to be a pain, but I just added another commit - that should be it now until testing.

Freshrojek commented:
Looks really good, James mentioned about the footer on dark mode so worth considering

tbentham commented:
Yeah, the red footer on dark mode is a bit glaring. I've now updated it to dark grey - with new screenshots to save you running the server. 😄

I also removed all instances of the `profile_image` field and added it to the exclude list in `forms.py`. (Resolves #276)

tbentham commented:
Just added another commit but that should be it now - I'm not planning to add anymore.

Code is already up-to-date with `master` so no need for rebasing (currently).

tbentham commented:
The PR was blocked by an unsigned commit from @geekzonebot so I had to rebase - will have to be approved yet again, sorry. 😓

tbentham commented:
Would someone mind approving this? Nothing's changed since the last review so it's ready to go - just needs a thumbs up.

Freshrojek commented:
Yeah sure, is it rebased with the pr sam just merged into master?

SamWinterhalder commented:
I shall rebase - 1 mo

SamWinterhalder reviewed (approved):
Mate, this is looking MUCH better. Congrats

Freshrojek commented:
> 
> 
> Mate, this is looking MUCH better. Congrats

I agree, makes a huge difference for users
